README.old.md

```diff
@@ -0,0 +1,2 @@
+# Covid Tracker App
+ 
```

Comment:
- [ ] Kindly remove this old readme file, since you have another one already created.